textx/model.py

```diff
@@ -658,8 +658,13 @@ def call_obj_processors(metamodel, model_obj,
 
                 for m in models:
                     # TODO: what does this check?
-                    assert not m._tx_reference_resolver.parser._inst_stack
+                    assert not _getattr(m, "_tx_reference_resolver").parser._inst_stack
```

Comment:
@goto40 I saw this TODO comment in the code. This line make sure that we have processed all parsed instances kept on `_inst_stack` stack. If at this point there is any instance on the stack we did something wrong.